README.md

```diff
@@ -49,29 +76,6 @@ DESCRIPTION
 
 _See code: [@oclif/plugin-help](https://github.com/oclif/plugin-help/blob/v5.2.7/src/commands/help.ts)_
 
-## `create-saas-boilerplate init PATH`
```

Comment:
@mkleszcz Running `oclif readme` removed this section. I decided to pretty much reuse it in the `Initialize new SaaS Boilerplate project` section